distribution/src/root/config/hazelcast-hardened.yaml

```diff
@@ -0,0 +1,233 @@
+# This is a sample Hazelcast configuration focused on hardened security.
+hazelcast:
+  # Custom cluster name protects against unintended cluster formation between foreign members.
+  cluster-name: ${custom.cluster.name}
+  properties:
+    # Don't listen on all network interfaces, but bind to just one.
+    hazelcast.socket.bind.any: false
+
+  management-center:
+    # Explicitly disable the scripting operations sent from the Management center.
+    # The default value is false already so this one is just to be sure and explicit.
+    scripting-enabled: false
+
+  user-code-deployment:
+    # Explicitly disable the user-code deployments. I.e. allow only code which already is on the classpath.
+    # The default value is false already so this one is just to be sure and explicit.
+    enabled: false
+
+  jet:
+    # If you don't need stream/batch processing features, leave them disabled.
```

Comment:
we can mention that it is disabled by default here too after https://github.com/hazelcast/hazelcast/pull/19000
also the resource uploading below